Marlin/Conditionals_post.h

```diff
@@ -618,6 +618,15 @@
   #endif
   #if HAS_HEATER_BED
     #define WRITE_HEATER_BED(v) WRITE(HEATER_BED_PIN, v)
+    #ifndef MAX_BED_POWER
+      #define MAX_BED_POWER 255
+    #endif
+    #ifndef HEATER_BED_ON
+      #define HEATER_BED_ON HIGH
+    #endif
+    #ifndef HEATER_BED_OFF
+      #define HEATER_BED_OFF LOW
+    #endif
```

Comment:
I hate to do this, but for consistency, it should probably be:
```cpp
#ifndef HEATER_BED_INVERTING
  #define HEATER_BED_INVERTING false
#endif
```
…and then code should use…
```cpp
WRITE_HEATER_BED( HEATER_BED_INVERTING); // Bed off
WRITE_HEATER_BED(!HEATER_BED_INVERTING); // Bed on
```
Alternatively, rename `WRITE_HEATER_BED` to `BED_HEATER_ON(v)` and change that macro's behavior based on an inverted bed pin.

The reason I recommend this approach is that it allows for other machines to also set an inverted bed pin, if they happen to have a similar kit, and it makes the inverted bed pin a general option that Hephestos 2 happens to enable, rather than something that refers back to Hephestos 2 in particular.